.github/workflows/test-linux.yml

```diff
@@ -0,0 +1,44 @@
+name: Test on Linux
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
should support [Ubuntu 20.04.5 LTS (Focal Fossa)](https://releases.ubuntu.com/focal/)

Reply:
upto [ubuntu-22.04.1  ](https://github.com/actions/runner-images) is available now

Reply:
should work!